spark/v3.5/spark/src/main/java/org/apache/iceberg/spark/actions/RewriteTablePathSparkAction.java

```diff
@@ -710,7 +710,7 @@ private boolean fileExist(String path) {
     return table.io().newInputFile(path).exists();
   }
 
-  private static String newPath(String path, String sourcePrefix, String targetPrefix) {
+  static String newPath(String path, String sourcePrefix, String targetPrefix) {
```

Comment:
its a bit messy but could we just use the util method in the test to avoid having to expose yet another method?

Reply:
sounds good, I revert this and used util method in test